docs/atlascli/command/atlas-deployments-start.txt

```diff
@@ -0,0 +1,92 @@
+.. _atlas-deployments-start:
+
+=======================
+atlas deployments start
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Start a deployment
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas deployments start <deploymentName> [options]
+
+.. Code end marker, please don't delete this comment
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - deploymentName
+     - string
+     - true
+     - Name of the deployment.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for start
+   * - --projectId
+     - string
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+   * - --type
+     - string
+     - false
+     - Type of the deployment that you want to create. Valid values are ATLAS or LOCAL.
```

Comment:
Delete "the".

`Type of deployment that you want to create. Valid values are ATLAS or LOCAL.`